yarn.lock

```diff
@@ -789,6 +789,11 @@
   resolved "https://registry.yarnpkg.com/@humanwhocodes/gitignore-to-minimatch/-/gitignore-to-minimatch-1.0.2.tgz#316b0a63b91c10e53f242efb4ace5c3b34e8728d"
   integrity sha512-rSqmMJDdLFUsyxR6FMtD00nfQKKLFb1kv+qBbOVKqErvloEIJLo5bDTJTQNTYgeyp78JsA7u/NPi5jT1GR/MuA==
 
+"@humanwhocodes/module-importer@^1.0.1":
```

Comment:
@dbjorge, I'd love to get your thoughts on this module. The first commit was about a month ago and only has 1 human contributor. That would probably be a red flag if we were consuming it directly as a shipping dependency; maybe it's less crucial when it's a transitive dev-dependency?

Reply:
It's definitely not thrilling, but we already have other transitive packages with bus factor 1 on this developer, and eslint is a well-maintained project that can handle forking/removing it if it becomes a problem. I think this is fine.